sdcm/sct_events/setup.py

```diff
@@ -84,6 +86,7 @@ def stop_events_device(_registry: Optional[EventsProcessesRegistry] = None) -> N
         EVENTS_HANDLER_ID,
         EVENTS_ANALYZER_ID,
         EVENTS_MAIN_DEVICE_ID,
+        EVENTS_COUNTER_ID,
```

Comment:
I think it should go before `EVENTS_MAIN_DEVICE_ID`, possibly before EVENTS_HANDLER_ID.